javascript/linked-list/stacks-and-queues/stack-queue-pseudo.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+const { Node, Stack } = require('./stacksandqueues');
+
+class PsuedoQueue {
+  constructor(value, next = null) {
+    this.value = value;
+    this.next = next;
+    this.s1 = new Stack();
+    this.s2 = new Stack();
+  }
+
+  enqueue(value) {
+    let node = new Node(value);
+    while (this.s1.top !== null)
+    {
+      this.s2.push(this.s1.pop());
+    }
+    this.s1.push(node);
+    while (this.s2.top !== null)
+    {
+      this.s1.push(this.s2.pop());
+    }
```

Comment:
Also, your pop method returns a node, so you need to extract the value of that node before you can push it to the other stack.